feature/preview/src/test/java/com/google/jetpackcamera/feature/preview/PreviewViewModelTest.kt

```diff
@@ -70,6 +73,13 @@ class PreviewViewModelTest {
         )
     }
 
+    @Test
+    fun captureImage() = runTest(StandardTestDispatcher()){
```

Comment:
Do we need a new test module to inject the test scope/dispatcher into the ViewModel?